src/Mezzurite.ts

```diff
@@ -140,35 +140,47 @@ export class MezzuritePlugIn implements ITelemetryPlugin{
         //     customProperties.systemTiming = JSON.stringify( e.SystemTiming);
         // }
         if (e.Timings && e.Timings.length > 0) {
+            var isRedirect  = -1;
+
             for (let i = 0; i < e.Timings.length; i++) {
                 let obj = e.Timings[i];
                 let metricType = obj.metricType.toString();
-                let loadTime = obj.value;
+
+                let props = {};
+                let measurements = {};
+
+                if (metricType === "Redirect") {
+                    isRedirect  = obj.value;
+                }
+                else {
+                    props["Redirect"] = isRedirect;
+                    measurements[metricType] = obj.value;
+                    this._appInsights.trackEvent({ name: "mz", properties: props, measurements: measurements});
+                }
 
                 if (obj.data) {
                     let componentTimes = JSON.parse(obj.data);
                     for (let j = 0; j < componentTimes.length; j++) {
                         let ct = componentTimes[j];
-                        let props = {};
-                        let measurements = {};
-                        props["metricType"] = metricType;
+                        let cprops = {};
+                        let cmeasurements = {};
+                        cprops["metricType"] = metricType;
                         let name = undefined;
                         let id = undefined;
```

Comment:
not used anymore?

Reply:
still used, I just updated the var names so I knew which were in the foreach.

Reply:
need to keep the metricTypeName here so we know which set of component timings we are looking at, all or vlt.